copy-and-push.js

```diff
@@ -0,0 +1,7 @@
+const copyAndPush = (num) => {
```

Comment:
Great job on this one, especially remembering to copy the array before manipulating it. 

One thing of note, if you look at your function in your test, you are passing in two parameters but when you create your function here, you are only requiring one argument, the num(array). You want to make sure that when you call a function, you pass in the same parameters as you set up originally when defining the function. 

This would be `copyAndPush = (arr, num)` and when you call it, you'd pass the parameters like so `copyAndPush(arr, num)` and then use those as template literals within your function declaration.